index.html

```diff
@@ -1,657 +1,1338 @@
-                        <a href="" data-cmp-clickable="" class="social-link-item">
-                            <svg width="23" height="23" viewBox="0 0 23 23" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                <path d="M3.04167 0.625C1.70646 0.625 0.625 1.70646 0.625 3.04167V19.9583C0.625 21.2935 1.70646 22.375 3.04167 22.375H19.9583C21.2935 22.375 22.375 21.2935 22.375 19.9583V3.04167C22.375 1.70646 21.2935 0.625 19.9583 0.625H3.04167ZM3.04167 3.04167H19.9583V19.9583H3.04167V3.04167ZM6.39998 4.63232C5.36444 4.63232 4.74325 5.25482 4.74325 6.08374C4.74325 6.91266 5.36452 7.5328 6.29614 7.5328C7.33168 7.5328 7.95288 6.91266 7.95288 6.08374C7.95288 5.25482 7.33161 4.63232 6.39998 4.63232ZM4.82585 9.08333V17.5417H7.875V9.08333H4.82585ZM10.3908 9.08333V17.5417H13.4399V12.9184C13.4399 11.5421 14.4219 11.3442 14.7167 11.3442C15.0115 11.3442 15.8 11.64 15.8 12.9184V17.5417H18.75V12.9184C18.75 10.2637 17.5703 9.08333 16.095 9.08333C14.6196 9.08333 13.8327 9.5746 13.4399 10.2633V9.08333H10.3908Z" fill="#9C9FA1"/>
-                            </svg>
-                        </a>
-                    </div>
-                </div>
-            </div>
-            <div class="footer-bottom-container">
-                <div class="inner-wrap">
-                    <div class="copy-right">
-                        <p class="content">©1994-2022. LeaderImpact™ All Rights Reserved.</p>
-                        <img srcset="./assets/copyright-ecfa.svg" src="./assets/copyright-ecfa.svg" loading="lazy" class="logo-img" itemprop="contentUrl"  alt="logo" title="logo">
-                    </div>
-                    <div class="term-policy-wrap">
-                        <span class="term-policy-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">Privacy</a>
-                        </span> 
-                        <span class="term-policy-item" data-cmp-data-layer="">
-                            <a data-cmp-clickable="" class="nav-link-item" href="">Term of Use</a>
-                        </span> 
+              </div>
+
+              <div class="container responsivegrid footer-bottom-container aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="container-48cdc20019" class="cmp-container">
+                  <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                    <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                      <div id="container-4873689adb" class="cmp-container">
+                        <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 inner-wrap">
+                          <div class=" responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="container-11ba88df40" class="cmp-container">
+                              <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 copy-right">

```

Comment:
This `inner-wrap` class either needs to move up 2 divs (with the `responsivegrid`) or a new nested container needs to be added below this one.